packages/ra-ui-materialui/src/layout/LoadingIndicator.js

```diff
@@ -13,6 +13,7 @@ const useStyles = makeStyles(
         loader: {
             margin: 14,
         },
+        icon: {},
```

Comment:
By reading the word `icon`, we can believe that we'll change the loading indicator. We should probably rename it to `loadedIcon` in order to be more explicit.

Reply:
Done!